tools/schemapi/schemapi.py

```diff
@@ -1226,6 +1241,12 @@ def _is_list(obj: Any | list[Any]) -> TypeIs[list[Any]]:
     return isinstance(obj, list)
 
 
+def _is_iterable(
+    obj: Any, *, exclude: type | tuple[type, ...] = (str, bytes)
```

Comment:
`(str, bytes)` chosen as they are the remaining `Iterable` types rom https://github.com/vega/altair/pull/3237#issuecomment-1806868507

`dict` is handled in an earlier case and `type` isn't `Iterable`

Reply:
I see, since the solution in this PR prevents the conversion to a list of byte strings it is preserving the current behavior which seems like a good move instead of introducing a change here. Passing a byte string will result in an error due to the schema validation like you mentioned and I think that is suitable.